src/lib/features/project/project-owners-read-model.test.ts

```diff
@@ -64,6 +118,12 @@ afterEach(async () => {
 });
 
 describe('integration tests', () => {
+    test('returns an empty list if there are no projects', async () => {
```

Comment:
We changed the return type here, so let's update the test name to match

```suggestion
    test('returns an empty object if there are no projects', async () => {
```